### DISCUSSION TITLE
3주차 과제 - (Poster) Design With Code

### DISCUSSION THREAD

**Discussion by youngsangcho** [youngsangcho/generativedesign2_2021#11] [Show and tell]
(과거의 / 상상의) 전시, 이벤트, 영화 등의 정보를 소개하는 포스터 디자인

* https://editor.p5js.org/ 사용
도형, 이미지, 텍스트 등을 사용

* 모션 / 인터랙션을 직접 구현하거나
아이디어를 글로 표현

+ 1개 이상의 스크린샷 이미지
+ (모션이나 인터랙션 있다면) 스크린 동영상 캡쳐하여 유튜브 업로드 유튜브 embed

+ 간단한 설명 (왜 / 어떻게)
+ p5js editor 의 File - Share - Present 링크
+ 코드 복사 붙여넣기

**Comment by SeyounJung01** [comment]
정세윤
https://blog.naver.com/seyounjj2001/222526574269

**Reply by eunjinhong** [reply]
홍은진
https://blog.naver.com/tnekf1012/222526598043

**Comment by Joeun410** [comment]
김조은

https://blog.naver.com/PostView.naver?blogId=joanne3456&Redirect=View&logNo=222526603118&categoryNo=10&isAfterWrite=true&isMrblogPost=false&isHappyBeanLeverage=true&contentLength=21043

**Comment by imyeeunok** [comment]
김예은

https://blog.naver.com/010312kye/222526616484

**Comment by KimHyunJuhg** [comment]
김현정
https://blog.naver.com/khj317888/222526634497

**Comment by jwejwepark** [comment]
https://glitch.com/edit/#!/pleasant-royal-paprika?path=sketch.js%3A26%3A66
수정본 같이 올리겠습니다
